### ISSUE TITLE
Emmet on VSCode does not resolve for shorthands with multiple curly braces

### ISSUE THREAD

**Issue by MeCode4Food** [microsoft/vscode#127919]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version:
- OS Version:

Steps to Reproduce:

1. Open any VSCode window
2. In the editor, type `div{{{test}}}`, and press ctrl + space to see the list of autocomplete suggestions
3. Autocomplete suggestions do not include the option to resolve emmet shorthand to `<div>{{test}}</div>`

![image](https://user-images.githubusercontent.com/25502412/124351877-818ce400-dc2f-11eb-9c1d-dd6ad5150185.png)

Other emmet features work as expected

**Comment by rzhao271** [comment]
Looks like we have to escape the braces before passing them over.

**Comment by MeCode4Food** [comment]
Yes, apparently the way to do it is by escaping the closing braces, as answered in this stack overflow question: https://stackoverflow.com/questions/68235276/how-do-you-escape-curly-braces-in-emmet-in-vscode
